policyengine_us/parameters/gov/states/me/tax/income/deductions/itemized/cap.yaml

```
@@ -0,0 +1,21 @@
+description: Maine caps its itemized deductions before medical and dental expenses at this amount.
+values:
+  2013-01-01: 27_500
+  2014-01-01: 27_950
+  2015-01-01: 28_350
+  2016-01-01: 28_450
+  2017-01-01: 28_600
+  2018-01-01: 29_050
+  2019-01-01: 29_550
+  2020-01-01: 30_050
+  2021-01-01: 30_400
+  2022-01-01: 31_150
+metadata:
+  unit: currency-USD
+  period: year
+  label: Maine itemized deduction pre medical and dental expenses cap
+  reference:
+    - title: Maine legislative code §5125. Itemized deductions
```

Could you add references for other years? To this and `phase_out.start`
```suggestion
    # Legal code specifies $28,350 as of 2015.
    - title: Maine legislative code §5125(4). Itemized deductions
```